modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/util/ExportImportObjectDefinitionUtil.java

```diff
@@ -27,6 +36,33 @@ public static void apply(
 			"objectLayoutColumns");
 	}
 
+	public static void prepareObjectDefinition(
```

Comment:
I was thinking about this method name :thinking: 
Someone may doubt whether we are preparing for the import or the export...
I found this example in the portal: `prepareLocalizedFieldsForImport(...)`. 
What do you think about renaming it to `prepareObjectDefinitionForImport`?